### DISCUSSION TITLE
INA219

### DISCUSSION THREAD

**Discussion by Dmkrd** [earlephilhower/arduino-pico#2240] [General]
Good afternoon!
please tell me how I can add sensor support INA219?
Thanks a lot!

**Comment by Andy2No** [comment]
See: https://learn.adafruit.com/adafruit-ina219-current-sensor-breakout

**Comment by lyusupov** [comment]
https://github.com/adafruit/Adafruit_INA219

**Comment by Dmkrd** [comment]
thanks to all! works!!! for a copy of VCC-GND YD RP2040, 4 and 5 pins of the board are used as SLC and SDA, and power is supplied to the sensor...
